Sources/AsyncHTTPClient/HTTPHandler.swift

```diff
@@ -237,9 +239,27 @@ extension HTTPClient {
         ///     - status: Response HTTP status.
         ///     - headers: Reponse HTTP headers.
         ///     - body: Response body.
+        @available(*, deprecated, message: "please use init(host:status:version:headers:body:) instead")
```

Comment:
there's a `renamed: init(host:status:version:headers:body:)` which gives a better error message and makes Xcode generate fixits automatically

Reply:
thanks, done!